torch/quantization/_generated/QAddGen.py

```diff
@@ -0,0 +1,33 @@
+
+# This file is generated using `torch.nn.quantization.make_module`
+# and saved as `QAddGen.py`.
+
+import torch
+
+r"""AddGen wraps the torch.add function."""
+class AddGen(torch.nn.Module):
```

Comment:
what's the point of calling it AddGen and then renaming it? Let's just give it the right name from the beginning